m2cgen/assemblers/boosting.py

```diff
@@ -60,13 +67,24 @@ def _assemble_bin_class_output(self, trees):
             proba_expr
         ])
 
-    def _assemble_single_output(self, trees, base_score):
-        trees_ast = [self._assemble_tree(t) for t in trees]
-        result_ast = utils.apply_op_to_expressions(
-            ast.BinNumOpType.ADD,
-            ast.NumVal(base_score),
-            *trees_ast)
-        return ast.SubroutineExpr(result_ast)
+    def _assemble_tree(self, node):
```

Comment:
So do we assemble tree? Or node? :) Let's rename the argument.

Reply:
Done